defuse/src/contract/tokens/nep141/deposit.rs

```diff
@@ -91,15 +92,15 @@ impl Contract {
     pub fn ft_resolve_deposit(
         &mut self,
         receiver_id: &AccountId,
-        token_ids: CoreTokenId,
-        deposited_amounts: u128,
+        token_ids: TokenId,
+        deposited_amounts: U128,
```

Comment:
Let's make mistakes less likely with:
```suggestion
        token: AccountId,
        amount: U128,
```

Reply:
Same for nft/mt_resolve_deposit: make it impossible to compile the code with the wrong token type